packages/@sfx/core/test/unit/common/index.test.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from '../../utils';
+import {
+  Core as CoreExport,
+  Plugin as PluginExport,
+  PluginMetadata as PluginMetadataExport,
+  PluginRegistry as PluginRegistryExport,
+} from '../../../src';
+import Core from '../../../src/core';
+import { Plugin, PluginMetadata, PluginRegistry } from '../../../src/plugin';
+
+describe('Entry point', () => {
+  it('should export Core', () => {
+    expect(CoreExport).to.equal(Core);
+  });
+
+  it('should export the Plugin interface', () => {
+    // The following should cause a compiler error if the two interfaces are not equal.
+    const plugin_is_assignable_to_plugin_export: PluginExport = {} as Plugin;
+    const plugin_export_is_assignable_to_plugin: Plugin = {} as PluginExport;
+  });
+
+  it('should export the PluginMetadata interface', () => {
+    // The following should cause a compiler error if the two interfaces are not equal.
+    const plugin_metadata_is_assignable_to_plugin_metadata_export: PluginMetadataExport = {} as PluginMetadata;
+    const plugin_metadata_export_is_assignable_to_plugin_metadata: PluginMetadata = {} as PluginMetadataExport;
+  });
+
+  it('should export the PluginRegistry interface', () => {
+    // The following should cause a compiler error if the two interfaces are not equal.
+    const plugin_registry_is_assignable_to_plugin_registry_export: PluginRegistryExport = {} as PluginRegistry;
+    const plugin_registry_export_is_assignable_to_plugin_registry: PluginRegistry = {} as PluginRegistryExport;
```

Comment:
Yeah, I think this is a good use case for comments. This feels very noisy, and I found the variable names confused me more than anything.

All of that said, we're testing an index file here. The actual value in the tests feels limited to me, so I don't want to spend too much time on it.

Reply:
If I add a helper function to assert that two interfaces are equal, I think it'll make the test much clearer.